src/sentry/event_manager.py

```diff
@@ -933,20 +943,19 @@ def get_event_type(data):
     return eventtypes.get(data.get("type", "default"))()
 
 
-def materialize_metadata(data, for_group=False, inject_metadata=None):
+def materialize_metadata(data, event_type, event_metadata):
```

Comment:
Are there any cases where the `event_type` passed into this function is different from `get_event_type(data)`? If not, `event_type` would not have to be in the signature.

Reply:
no, but you don't want to compute it twice in `_materialize_metadata_many`, that's the whole reason. And I don't want to add more branches inside.